src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -868,14 +868,23 @@ def __call__(
         else:
             guidance = None
 
+        # TODO: Clarify this section
```

Comment:
```suggestion
        # TODO: Clarify this section
```
Will let @yiyixuxu comment, lgtm though